ci/nvim-lspconfig/default.nix

```diff
@@ -19,7 +19,14 @@ let
     ];
   };
 
-  nvim = wrapNeovimUnstable neovim-unwrapped nvimConfig;
+  # TODO: 2025-01-06
+  # Wait for user feedback on disabling the fixup phase.
+  # Ideally this will be upstreamed to nixpkgs.
+  # See https://github.com/nix-community/nixvim/pull/3660#discussion_r2326250439
```

Comment:
```suggestion
```


Sorry I wasn't clear earlier. Because this is in the `ci/` folder, we shouldn't need to consider end users here. These are scripts to be run internally, usually by GHA.